DungeonSource/Dungeon.java

```diff
@@ -45,23 +45,24 @@
 */
 public class Dungeon
 {
+	private static Scanner kb = new Scanner(System.in);
+	
     public static void main(String[] args)
 	{
-    	System.out.println("Choose a hero:\n" +
-			       "1. Warrior\n" +
-				   "2. Sorceress\n" +
-				   "3. Thief");
-    	Scanner kb = new Scanner(System.in);
-    	int random = (int)(Math.random() * 3) + 1;
-
 		do
 		{
+			System.out.println("Choose a hero:\n" +
```

Comment:
Moved prompt to inside do/while loop since we would only print the prompt once before refactoring